our-brothers-web/src/app/shell/header/header.component.scss

```diff
@@ -12,7 +12,7 @@ $mobileBreakpoint: 1200px;
 
     %bar {
       width: 30px;
-      height: $barHeight;
```

Comment:
Can we update `$barHeight` value to `3px`?

Reply:
no, If you change the parameter the interval will also change which is not good, we only need to change the height of the line